docs/en/reference/query-builder.rst

```diff
@@ -51,8 +51,8 @@ Building a Query
 ----------------
 
 The ``\Doctrine\DBAL\Query\QueryBuilder`` supports building ``SELECT``,
-``INSERT``, ``UPDATE`` and ``DELETE`` queries. Which sort of query you
-are building depends on the methods you are using.
+``INSERT``, ``UPDATE``, ``DELETE`` and ``UNION`` queries. Which sort of
+query you are building depends on the methods you are using.
```

Comment:
There is no `UNION` query in SQL. It's technically still a `SELECT`.

Reply:
Should we avoid naming it differntly only for the documentation, or also avoid the QueryType::UNION enum case ? But than a additional flag would be required to determine that a union should be build.

Reply:
_How_ we implement unions internally is an implementation detail at this point. This piece of documentation is a high level introduction that explains the purpose of the QB to someone who is familiar with SQL. Mentioning `UNION` here is simply confusing.

Reply:
Reverted that part, but added a `UNION` clause section. Hope that is better.